app/src/main/java/com/antosik/benchproject/app/popular/movies/view/PopularMoviesFragment.kt

```diff
@@ -23,7 +23,7 @@ class PopularMoviesFragment : Navigator<PopularMoviesFragmentNavigator>() {
         val binding = PopularMoviesFragmentBinding.inflate(inflater, container, false)
```

Comment:
You should use `apply` is such cases 😄

Reply:
To not repeat `binding...`

Reply:
And thanks to that you can even use `=` instead of `{}`

Reply:
I changed it